README.md

```diff
@@ -3,100 +3,36 @@
 [![PyPI version](https://badge.fury.io/py/aiida-cp2k.svg)](https://badge.fury.io/py/aiida-cp2k)
 
 # AiiDA CP2K
-**The CP2K plugin for the AiiDA workflow and provenance engine.**
+AiiDA plugin for CP2K.
 
 
-![logo](./aiida-cp2k_logo.png)
-
-# Installation
+## Installation
 
 If you use `pip`, you can install it as: 
 ```
 pip install aiida-cp2k
 ```
 
-
-# Features
-Following the philosophy to _''enable without getting in the way''_, this plugin provides access to all of CP2K's capabilities through a small set of well-tested features:
-
-- A full [CP2K input](https://manual.cp2k.org) has to be provided as a nested Python dictionary ([example](./test/test_dft.py)):
+To install the plugin in an editable mode, run:
 ```
-params = {'FORCE_EVAL': {'METHOD': 'Quickstep', 'DFT': { ... }}}
-calc.use_parameters(ParameterData(dict=params))
+git clone https://github.com/aiidateam/aiida-cp2k
+cd aiida-cp2k
+pip install -e .  # Also installs aiida, if missing (but not postgres/rabbitmq).
 ```
 
-- Section parameters are stored as key `_` in the dictionary:
-```
-xc_section = {'XC_FUNCTIONAL': {'_': 'LDA'}}
-```
+## Links
+* [Documentation](https://aiida-cp2k.readthedocs.io/en/latest/) for the calculation examples and features of the plugin.
+* [Make an issue](https://github.com/aiidateam/aiida-cp2k/issues/new) for bug reports, questions and suggestions.
+* [AiiDA](http://www.aiida.net/) to learn about AiiDA.
```

Comment:
aiida & cp2k you could already link in the text directly

Reply:
ok